clang/include/clang/Driver/Options.td

```diff
@@ -1980,9 +1980,11 @@ within a command line are combined with spaces; spaces and backslashes within an
 argument are escaped with backslashes. This format differs from the format of
 the equivalent section produced by GCC with the -frecord-gcc-switches flag.
 This option is currently only supported on ELF targets.}]>,
-  Group<f_clang_Group>;
```

Comment:
Will the DocBrief section above lead to mentions of Clang in https://flang.llvm.org/docs/FlangCommandLineReference.html?

Reply:
Nice catch. The text only contains a single reference to clang. I have removed it, so it should apply to both `clang` and `flang` now.